app/src/main/java/com/fantastic/bookxchange/fragments/AddBookSearchingFragment.java

```diff
@@ -0,0 +1,197 @@
+package com.fantastic.bookxchange.fragments;
+
+import android.content.Context;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v7.widget.SearchView;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ListView;
+import android.widget.ProgressBar;
+import android.widget.Toast;
+
+import com.fantastic.bookxchange.R;
+import com.fantastic.bookxchange.adapters.BookTestAdapter;
+import com.fantastic.bookxchange.api.BookClient;
+import com.fantastic.bookxchange.api.JsonKeys;
+import com.fantastic.bookxchange.models.BookTest;
+import com.loopj.android.http.JsonHttpResponseHandler;
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.util.ArrayList;
+
+import cz.msebera.android.httpclient.Header;
+import retrofit2.http.PUT;
+
+
+public class AddBookSearchingFragment extends Fragment {
+
+    private ListView lvBooks;
```

Comment:
Better recycler view for this! Actually, I think you can probably reuse the base fragment I created for the books :)

Reply:
Yes, I'll fix it